File: Python/Ex15.py
print("Com esse programa sera calculado seu IMC")
a= float(input("Digite seu peso em 'KG': "))
b= float(input("Digite sua altura em 'm': "))
imc=a/(b*b)
print(f"Seu IMC eh: {imc:.2f}")
if (imc<20):
    print("Abaixo do peso!")
elif (imc>=20) and (imc<25):
    print("Peso ideal!")
else:
     print("Acima do peso!")       

File: C/Ex15.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
    float p, h, imc;

    printf("Digite seu peso: ");
    scanf("%f", &p);
    printf("Digite sua altura: ");
    scanf("%f", &h);
    imc=p/(h*h);
    if (imc<20) {
        printf("Abaixo do peso = %.2f", imc);
        }
    else if(20<=  imc <25) {
        printf("Peso ideal = %.2f", imc);
        }
    else if(imc>=25) {
        printf("Acima do peso = %.2f", imc);
        }

        return 0;

        }    